crates/devnet/src/types.rs

```diff
@@ -183,7 +181,6 @@ pub async fn load_config() -> ParsedConfig {
     }
 }
 
-/// This module is needed until [(#710)](https://github.com/near/mpc/issues/710) is resolved.
```

Comment:
it seems to me that some functions here are still needed. Could you clarify @kevindeforth ?